src/frontend/src/routes/router.tsx

```diff
@@ -4,11 +4,25 @@ import LandingPage from '../pages/LandingPage/LandingPage';
 import PPQConnectPage from '../pages/PPQConnectPage';
 import PPQFormPage from '../pages/PPQFormPage/PPQFormPage';
 import PPQLandingPage from '../pages/PPQPage/PPQPage';
+import {
+  getItemFromStorage,
+  getPIAIntakeFormFlagFromStorage,
+} from '../utils/helper.util';
 
 const Router = () => {
   return (
     <Routes>
-      <Route path="/ppq" element={<PPQLandingPage />} />
+      <Route
+        path="/ppq"
+        element={
+          getItemFromStorage('config') &&
+          getPIAIntakeFormFlagFromStorage() === true ? (
+            <PPQLandingPage pia={true} />
+          ) : (
+            <PPQLandingPage pia={false} />
```

Comment:
Also, please change to proper naming convention of the prop name, like `enablePiaIntakeForm`